state/sqlserver/metadata.yaml

```diff
@@ -0,0 +1,101 @@
+# yaml-language-server: $schema=../../../component-metadata-schema.json
+schemaVersion: "v1"
+type: "state"
+name: "sqlserver"
+version: "v1"
+status: "stable"
+title: "SQL Server"
+urls:
+  - title: "Reference"
+    url: "https://docs.dapr.io/reference/components-reference/supported-state-stores/setup-sqlserver/"
+capabilities:
+  # If actorStateStore is present, the metadata key actorStateStore can be used
+  - "actorStateStore"
+  - "crud"
+  - "transactional"
+  - "etag"
+authenticationProfiles:
+  - title: "Connection string"
+    description: |
+      Authenticate using a connection string.
+    metadata:
+      - name: connectionString
+        required: true
+        sensitive: true
+        description: |
+          The connection string used to connect.
+          If the connection string contains the database it must already exist. Otherwise, if the database is omitted, a default database named "Dapr" is created.
+        example: |
+          "Server=myServerName\myInstanceName;Database=myDataBase;User Id=myUsername;Password=myPassword;"
+builtinAuthenticationProfiles:
+  - name: "azuread"
+    metadata:
+      - name: useAzureAD
+        required: true
+        type: bool
+        description: |
+          Must be set to `true` to enable the component to retrieve access tokens from Azure AD.
+          This authentication method only works with Azure SQL databases.
+      - name: connectionString
+        required: true
+        sensitive: true
+        description: |
+          The connection string or URL of the Azure SQL database, without credentials.
+          If the connection string contains the database it must already exist. Otherwise, if the database is omitted, a default database named "Dapr" is created.
```

Comment:
```suggestion
          If the connection string contains the database, it must already exist. Otherwise, if the database is omitted, a default database named "Dapr" is created.
```